homeassistant/components/wallbox/number.py

```diff
@@ -72,12 +72,18 @@ def __init__(
         self._coordinator = coordinator
         self._attr_name = f"{entry.title} {description.name}"
         self._attr_unique_id = f"{description.key}-{coordinator.data[CHARGER_DATA_KEY][CHARGER_SERIAL_NUMBER_KEY]}"
+        self._is_quasar = coordinator.data[CHARGER_DATA_KEY][CHARGER_PART_NUMBER_KEY].startswith("QSX")
```

Comment:
lets call this something else (is_bidirectional?), I think there will be more chargers that support this in the future. Would be a shame if we would need to change the code then. Also, lets create a list of prefixes `['QSX','...']` and match the first 3 characters of the part number to these. This will allow more bidirectional chargers in the future.